extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -95,16 +96,24 @@ function render_player( $track_list, $attributes ) {
 	$block_classname = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attributes );
 
 	ob_start();
-
 	?>
-	<div class="<?php echo esc_attr( $block_classname ); ?>">
-		<?php // Placeholder: block markup is being handled in https://github.com/Automattic/jetpack/pull/14952. ?>
-		<script type="application/json" class="wp-playlist-script"><?php echo wp_json_encode( $player_data ); ?></script>
+	<div class="<?php echo esc_attr( $block_classname ); ?>" id="<?php echo esc_attr( $instance_id ); ?>">
+		<ol>
+			<?php
+			foreach ( $track_list as $attachment ) :
+				printf( '<li><a href="%1$s" data-jetpack-podcast-audio="%2$s">%3$s</a></li>', esc_url( $attachment['link'] ), esc_url( $attachment['src'] ), esc_html( $attachment['title'] ) );
+			endforeach;
+			?>
+		</ol>
+		<script type="application/json"><?php echo wp_json_encode( $player_data ); ?></script>
 	</div>
+	<script>window.jetpackPodcastPlayers=(window.jetpackPodcastPlayers||[]);window.jetpackPodcastPlayers.push( <?php echo wp_json_encode( $instance_id ); ?> );</script>
 	<?php
 	/*
 	* Enqueue necessary scripts and styles.
 	*/
+	wp_enqueue_script( 'wp-mediaelement' );
```

Comment:
Now that #14958 is merged, you should be able to add this as a dependency on https://github.com/Automattic/jetpack/pull/15008/files#diff-551fe0712dea509839f9d2b144489d57R117

Reply:
looking

Reply:
rebased, implemented and confirmed I can still see the player working thanks to the enqueued MediaElement via script dependency ✅
<img width="639" alt="Screenshot 2020-03-17 at 17 08 04" src="https://user-images.githubusercontent.com/156676/76876215-e4f3d680-6871-11ea-9122-41a716d65bcc.png">